src/App.vue

```diff
@@ -1350,8 +1350,6 @@ export default class App extends Mixins(CommonMixin, DateMixin, FilingTemplateMi
 
     if (addresses) {
       this.setOfficeAddresses(addresses)
-    } else {
-      throw new Error('Invalid office addresses')
```

Comment:
I was initially concerned because I thought this code was used for multiple filing types. However, I see that it's only called for restorations, so this is OK.

Can you please add a comment before line 1347 indicating that this method is only called for restorations?

Reply:
Thanks. Resolving.